app/views/blocked/users/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2 class="heading"><%= t(".title") %></h2>
+
+<div class="notice">
```

Comment:
This one also for the other reviewers: will this ever be different from the confirm_block page?

Reply:
Yeah, the confirm_block page is eventually going to have options on it for things like "delete comments" or "also mute" or whatever, so the text and formatting will diverge as time goes on.